collectors/proc.plugin/proc_net_wireless.c

```diff
@@ -0,0 +1,453 @@
+                                                                PLUGIN_PROC_NAME,
+                                                                PLUGIN_PROC_MODULE_NETWIRELESS_NAME,
+                                                                NETDATA_CHART_PRIO_WIRELESS_IFACE + 1,
+                                                                update_every,
+                                                                RRDSET_TYPE_LINE);
+                rrdset_flag_set(wireless_dev->st_link, RRDSET_FLAG_DETAIL);
+
+                wireless_dev->rd_link = rrddim_add(wireless_dev->st_link, "link", NULL, 1, 1,
+                                                   RRD_ALGORITHM_ABSOLUTE);
+            } else {
+                rrdset_next(wireless_dev->st_link);
+            }
+
+            if (unlikely(!wireless_dev->st_level)) {
+                wireless_dev->st_level = rrdset_create_localhost("wireless",
+                                                                 wireless_dev->chart_id_net_level,
+                                                                 NULL,
+                                                                 wireless_dev->name,
+                                                                 "wireless.quality_level",
+                                                                 "How strong the received signal is.",
+                                                                 "dBm",
+                                                                 PLUGIN_PROC_NAME,
+                                                                 PLUGIN_PROC_MODULE_NETWIRELESS_NAME,
+                                                                 NETDATA_CHART_PRIO_WIRELESS_IFACE + 2,
+                                                                 update_every,
+                                                                 RRDSET_TYPE_LINE);
+                rrdset_flag_set(wireless_dev->st_level, RRDSET_FLAG_DETAIL);
+
+                wireless_dev->rd_level = rrddim_add(wireless_dev->st_level, "level", NULL, 1, 1,
+                                                    RRD_ALGORITHM_ABSOLUTE);
+            } else {
+                rrdset_next(wireless_dev->st_level);
+            }
+
+            if (unlikely(!wireless_dev->st_noise)) {
+                wireless_dev->st_noise = rrdset_create_localhost("wireless",
+                                                                 wireless_dev->chart_id_net_noise,
+                                                                 NULL,
+                                                                 wireless_dev->name,
+                                                                 "wireless.quality_noise",
+                                                                 "Background noise level when a package is not transmitted.",
+                                                                 "dBm",
```

Comment:
> wireless.quality_noise

i suggest:
 - `context`: wireless.noise_level
 - `title`: The noise level indicates the amount of background noise in your environment. The closer the value to 0, the greater the noise level.